### DISCUSSION TITLE
EvaluateScriptAsync on Windows 11

### DISCUSSION THREAD

**Discussion by JRDove** [cefsharp/CefSharp#4484] [General]
Hi,
It seems to me that EvaluateScriptAsync cannot be used on Windows 11.  Is that correct? ExecuteScriptAsync is not a problem so I have created a clunky workaround involving temporary files on a website. This works fairly well but it's not ideal.
JR Dove

**Comment by amaitland** [comment]
> It seems to me that EvaluateScriptAsync cannot be used on Windows 11. Is that correct?

`EvaluateScriptAsync` uses `CEF native IPC`, it should work on all supported platforms.

What version are you using?

What is the exception you are seeing?

**Reply by JRDove** [reply]
Thanks for your reply. 
Windows: Edition Family, Version: 22H2, Build: 22621.1555 Experience: Windows Feature Experience Pack 1000.22640.1000.0 Language French
CefSharp.Winforms 112.2.70 112.3.0 via NuGet.
UPDATE
You are right. It is not EvaluateScriptAsync causing the problem. I now have an older version (made with VS 2012) running and using EvaluateScriptAsync in  VS 2022. I'll try to convert it to the latest version of CefSharp, which is what I did last time, and see what happens.
UPDATE 2
The program is now functioning with the latest version of CefSharp in VS 2022 on Windows 11, albeit with some non cef-related bugs. I've no idea what went wrong before. Maybe I didn't delete all the previous version correctly. Sorry if you were worried but I guess you probably weren't :-)

**Comment by aimcosoftware** [comment]
@JRDove Our app uses a lot of calls to `EvaluateScriptAsync`,  we haven't had any issues with it running on Windows 11.
